scout/commands/export/variant.py

```diff
@@ -105,6 +105,41 @@ def verified(collaborator, test, outpath=None):
     return written_files
 
 
+@click.command("managed", short_help="Export managed variants")
+@click.option(
+    "-c",
+    "--collaborator",
+    help="Specify what collaborator to export variants from. Defaults to all variants.",
+)
+@click.option(
+    "-b",
+    "--build",
+    help="Specify what build to export variants from. Defaults to all variants.",
+)
+@json_option
+@with_appcontext
+def managed(collaborator: str, build: str, json: bool):
+    """Export causatives for a collaborator in .vcf format"""
```

Comment:
```suggestion
    """Export causatives for a collaborator in .vcf or json format."""
```

Reply:
Indeed! Only it is actually managed variants.. 😆

Reply:
We almost need the second reviewer 😆

Reply:
Hehe, the obnoxious and scary one! Just letting stuff sit for one day until we completely forget what it was about and rereading does wonders. 😆